dotnet/src/SemanticKernel.Abstractions/Orchestration/ContextVariables.cs

```diff
@@ -26,10 +26,26 @@ public sealed class ContextVariables : IDictionary<string, string>
     /// <summary>
     /// Constructor for context variables.
     /// </summary>
-    /// <param name="value">Optional value for the main variable of the context including trust information.</param>
-    public ContextVariables(string? value = null)
+    /// <param name="input">Optional value for the main variable of the context including trust information.</param>
```

Comment:
Is this "trust information" part of this comment still accurate?

Reply:
Nope. Will remove